components/inputs/input-date.js

```diff
@@ -297,14 +301,18 @@ class InputDate extends SkeletonMixin(FormElementMixin(LocalizeCoreElement(LitEl
 			if (prop === '_dateTimeDescriptor' || prop === 'value') {
 				this._shownValue = this.value;
 				this._setFormattedValue();
-			}
-			if (prop === 'value') {
-				this.setFormValue(this.value);
-				this.setValidity({
-					rangeUnderflow: !this.noValidateMinMax && this.value && this.minValue && getDateFromISODate(this.value).getTime() < getDateFromISODate(this.minValue).getTime(),
-					rangeOverflow: !this.noValidateMinMax && this.value && this.maxValue && getDateFromISODate(this.value).getTime() > getDateFromISODate(this.maxValue).getTime()
-				});
-				this.requestValidate(false);
+
+				if (prop === 'value') {
+					this.setFormValue(this.value);
+					this.setValidity({
+						rangeUnderflow: !this.noValidateMinMax && this.value && this.minValue && getDateFromISODate(this.value).getTime() < getDateFromISODate(this.minValue).getTime(),
+						rangeOverflow: !this.noValidateMinMax && this.value && this.maxValue && getDateFromISODate(this.value).getTime() > getDateFromISODate(this.maxValue).getTime()
+					});
+					this.requestValidate(false);
+				}
+			} else if (prop === 'opened') {
+				if (this.opened) this._open();
+				else if (!this.opened && this._dropdown.opened) this._close();
```

Comment:
Nit: the check for `this._dropdown.opened` could move into `_close()` just in case we want to call `_close()` from elsewhere maybe?